build/nsm/docker/Dockerfile

```diff
@@ -1,10 +1,15 @@
-FROM golang:1-alpine as build
+FROM golang:1 as build
```

Comment:
Just to be clear, these are Debian images? The apt commands below lead me to believe that, but perhaps busybox also uses apt.

Reply:
The build is on Debian stretch. The runtime is on busybox with the compiled version copied over.

Reply:
Thanks for clarifying! Now I have the same question as @edwarnicke :)